src/main/resources/view/MainWindow.fxml

```diff
@@ -55,7 +55,8 @@
                             <StackPane fx:id="personListPanelPlaceholder" VBox.vgrow="ALWAYS"/>
                         </VBox>
                     </VBox>
-                    <VBox minWidth="250" prefWidth="250" styleClass="pane-with-border">
+                    <VBox minWidth="250" maxWidth="250" prefWidth="250" styleClass="pane-with-border"
+                          VBox.vgrow="ALWAYS">
```

Comment:
VBox.vgrow ensures that the iterface should grow vertically to fill any available extra space